ted2zim/entrypoint.py

```diff
@@ -14,11 +14,29 @@ def main():
         prog=NAME, description="Scraper to create ZIM files from TED videos",
     )
 
+    parser.add_argument(
+        "--categories",
+        help="Comma-seperated list of topics to scrape. Should be exactly same as given on ted.com/talks",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--max-videos",
```

Comment:
`--max-videos-per-topic` would make it clearer

Reply:
Done. Renamed to --max-videos-per-topic